src/parser/parse.ts

```diff
@@ -150,9 +154,9 @@ const lex = (tokens: Token[]): DocNode[] => {
   return children
 }
 
-const parse = (str: string): DocNode[] => {
+const parse = (str: string, patternString: string = '{(\\w+)}'): DocNode[] => {
```

Comment:
before you also matched `-` but not now. Not sure if it's bad or not

Reply:
My initial thought was that `\w` more or less conforms with how JS variables can be, eg. `const foo_Bar` since `const foo-Bar` is not allowed. However since the data object is a `Record<string, x>` I guess users could technically do `{ 'foo-bar': 'baz' }`.

I will change it to `[\w-]+`